### ISSUE TITLE
Indicate external link behavior

### ISSUE THREAD

**Issue by aviannamiller** [omeka/plugin-LocalContexts#7]
https://github.com/omeka/Omeka/issues/1108 deals with external link accessibility in core, adding screen reader text and visible icon indicators to links that open in new tabs (or windows). Admin-side, Local Contexts external links pick up on the icon but not the screen reader text. On site pages, neither the icon or screen reader text are present and should be added in.
